base/logging_context.go

```diff
@@ -51,6 +67,11 @@ type DbConsoleLogConfig struct {
 	LogKeys  *LogKeyMask
 }
 
+// DbAuditLogConfig can be used to customise the audit logging for events associated with this database.
+type DbAuditLogConfig struct {
+	EnabledEvents map[AuditID]struct{}
```

Comment:
How does this intersect with default enabled events? Or is default enabled events something we are going to implement later?

Reply:
The set of default enabled events will make its way down to this field once wired up in [CBG-3975](https://issues.couchbase.com/browse/CBG-3975)